### PR TITLE
Add timestamp to title.

### PR DESCRIPTION
**Pull request by BenJamesBen**
This change adds a timestamp to the Title produced by plugin-title, and it also updates and fixes a typo in its documentation.

This pull request accomplishes part of Enhancement Request #2027. However, because it doesn't completely address all aspects of it (things like adding comments/notes to patterns), I am leaving #2027 open.

### CODE REVIEW

File: markdown/org/docs/various/notation/title/en.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Titles
 ---
 
-Each pattern piece has a title that tells you the number and name of the piece, as well as the pattern name:
+Each pattern piece has a title that tells you the number and name of the piece, as well as the pattern name and a timestamp of when the pattern was rendered:
```
Comment (joostdecock) [inline markdown/org/docs/various/notation/title/en.md:5]: Perhaps *when the pattern was **drafted*** is clearer to users? To render is kinda nerdy, not sure everyone will get what that means.

File: plugins/plugin-title/src/index.js [hunk 2]
```diff
@@ -92,6 +92,14 @@ export default {
           .attr('data-text-class', 'fill-current font-bold')
           .attr('data-text-transform', transform(so.at.shift(-90 - so.rotation, shift * so.scale)))
       }
+      shift += 8
+      const dateformat = require('dateformat')
+      const now = new Date()
+      this.points[`_${prefix}_exportDate`] = so.at
+        .shift(-90 - so.rotation, shift * so.scale)
+        .attr('data-text', dateformat(now, 'yyyymmdd"T"HHMMo'))
+        .attr('data-text-class', 'fill-note')
```
Comment (joostdecock) [inline plugins/plugin-title/src/index.js:101]: Could we make that class `text-sm` so it's a bit smaller and just black?

**Review by joostdecock — DISMISSED** [review]

File: markdown/org/docs/various/notation/title/en.md (hunk 1 above)
Comment (BenJamesBen) [reply markdown/org/docs/various/notation/title/en.md]: I made the suggested change.

File: plugins/plugin-title/src/index.js (hunk 2 above)
Comment (BenJamesBen) [reply plugins/plugin-title/src/index.js]: Change made.

**Comment by BenJamesBen** [comment]
I made the suggested changes. The title now looks like:
Title timestamp, smaller text

**Review by joostdecock — APPROVED** [review]

**Pull request merged**